Add PULL REQUEST TEMPLATE and CHANGELOG (#8)
vtex-apps/app-store-theme: .github/PULL_REQUEST_TEMPLATE.md

#### What problem is this solving?

<!--- What is the motivation and context for this change? -->

#### How should this be manually tested?

[Workspace](url)

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [ ] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Screenshots or example usage

> #### Type of changes
> 
> <!--- Add a ✔️ where applicable -->
> ✔️ | Type of Change
> ---|---
> _ | Bug fix <!-- a non-breaking change which fixes an issue -->
> _ | New feature <!-- a non-breaking change which adds functionality -->
> _ | Breaking change <!-- fix or feature that would cause existing functionality to change -->
> _ | Technical improvements <!-- chores, refactors and overall reduction of technical debt -->
> 
> #### Notes
> 
> <!-- Put any relevant information that doesn't fit in the other sections here. -->

Reviewer: ```suggestion
#### Type of changes

- [ ] Bug fix <!-- a non-breaking change which fixes an issue -->
- [ ] New feature <!-- a non-breaking change which adds functionality -->
- [ ] Breaking change <!-- fix or feature that would cause existing functionality to change -->
- [ ] Refactoring <!-- chores, refactors and overall reduction of technical debt -->

```

This just a personal preference, but I would:

- Checkboxes are easier to write/edit than tables and copying/paste the character ✔️
- "Technical improvements" sounds very vague to me and "Refactoring" is a pretty known term so I would prefer the later
- Since  `### Notes` section is optional by default I'd leave it commented out (avoiding PRs with an empty section) or leave to the proposer (PR opener) to write it

Author: Hmmm, I'm using a default template but theses changes look good to me.
